### ISSUE TITLE
Runners Version Incorrect

### ISSUE THREAD

**Issue by bigmstone** [StackStorm/st2#4210]
##### SUMMARY
Runners should have their version set automatically like the rest of ST2's components instead of hard coded (currently to `2.5.0`)

##### ISSUE TYPE
 - Bug Report

**Comment by Kami** [comment]
Yeah, we eventually need to move away from the way we currently install / use runners (mostly by manipulating `PYTHONPATH` when loading / instantiating a runner) to install them in the same way as any other Python dependency.

That means building wheels for those runner packages and then installing them into bundled st2 Python virtualenv during package build process.

All the runners are now Python packages so that should be possible. It will require some small amount of work in st2 (mostly removing loading code) and more work on the infra / building packages code.

I tagged it as 2.9.0 task since I would like to see that solved soon with the Orchestra issue we are having. Before orchestra it wasn't a priority, but now it is, so we can get rid of the hack / workaround in st2-packages post install script.
